autoscheduler/frontend/src/components/SchedulingPage/Schedule/Schedule.tsx

```diff
@@ -130,7 +131,10 @@ const Schedule: React.FC<RouteComponentProps> = () => {
       time2 = 8 * 60 + 0;
     } else if (evt.clientY > evt.currentTarget.getBoundingClientRect().bottom) {
       time2 = 21 * 60 + 0;
-    } else {
+    } else if (
+      // these conditions ensure that the time disappears if the mouse leaves to the side
+      evt.clientX >= evt.currentTarget.parentElement.getBoundingClientRect().left
```

Comment:
2 of them are actually for `currentTarget`, and 2 are actually for the `parentElement`. I think I'll go ahead and make local variables for readability/brevity anyway, but it might not help performance much